gst/nnstreamer/tensor_split/gsttensorsplit.c

```diff
@@ -673,6 +676,7 @@ gst_tensor_split_get_property (GObject * object, guint prop_id,
             strv = p;
           }
         }
+        /* the second parameter, strv (i.e., gchar *v_string), is nullable */
```

Comment:
consider to set empty string if strv is nullable
g_value_take_string (value, strv ? strv : "");

Reply:
Applied. Thanks.